leakpro/attacks/mia_attacks/lira.py

```diff
@@ -194,7 +214,7 @@ def run_attack(self:Self) -> CombinedMetricResult:
         score = np.asarray(score)  # Convert the list of scores to a numpy array
 
         # Generate thresholds based on the range of computed scores for decision boundaries
-        self.thresholds = np.linspace(np.min(score), np.max(score), 1000)
+        self.thresholds = np.linspace(np.nanmin(score), np.nanmax(score), 2000)
```

Comment:
Dont use nanmin and nanmax! It just hides bugs. If we handle the audit points missing in or out models as in rmia, we dont need this.

Reply:
Done